src/FixerConfiguration/AliasedFixerOption.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\FixerConfiguration;
+
+/**
+ * @author ntzm
+ *
+ * @internal
+ */
+final class AliasedFixerOption implements FixerOptionInterface
```

Comment:
Please add the following annotation is the DocComment of this class, `AliasedFixerOptionBuilder` and `AbstractFixerWithAliasedOptionsTestCase`: `@TODO on 3.0: drop this class`.